grep-9999-correctiq.md

```diff
@@ -0,0 +1,57 @@
+# GREP [XXXX] -- Upstreaming gr-correctiq
+
+- Original Author: Michael Piscopo <ghostop14@gmail.com>
+- Champion: Michael Piscopo <ghostop14@gmail.com>
+- Status: Draft
+
+History:
+- 07-Jan-2020: Initial Draft
+
+## Abstract
+
+The DC spike created by the IQ sampling process has long been an issue with the sampling 
+process. Software such as GQRX and SDRSharp have built-in algorithms that can be enabled 
+to remove this artifact.  However, GNU Radio has no built-in blocks to address 
+this feature.  A second related feature common in receiver software is the ability 
+to correct for inverted spectrum.  The gr-correctiq OOT module provides blocks that 
```

Comment:
I don't think this PR justifies a new in-tree component. The blocks, however, are valuable and interesting. They could go into gr-blocks.